client/blocks/image-editor/image-editor-buttons.jsx

```diff
@@ -41,25 +45,25 @@ class ImageEditorButtons extends Component {
 			<div className="image-editor__buttons">
 				{ onCancel && (
 					<Button
-						className="image-editor__buttons-button"
+						className="image-editor__buttons-button button"
```

Comment:
I'm not particularly fond of this solution. Instead, I'd prefer a more componentized way to do this. How about creating a wrapper component that adds this classname to the rest of the classnames? We could then use that component to do the migration and when we're done, we could remove that component altogether or replace it with the Calypso button component, which would itself be the wrapper.

Reply:
Yeah I don't like it either.

Are you suggesting we have something like this:

```js
const CalypsoButton => ( { className, ...restProps ) =>
    <WordPressButton
        className={classnames(className, 'button')
        {...restProps}
    />
```

Just a wrapper component? That seems fine to me... however I think I'm confused about this part of your suggestion:

> we could remove that component altogether or replace it with the Calypso button component, which would itself be the wrapper.

Reply:
🤔 I must have used a different English version than the one we usually use around here 😄 

Yeah, I was talking exactly about a wrapper component. And I meant that eventually, if possible without adding ambiguity, we could get rid of that wrapper component if the only thing it's doing is to add a classname to a component. This could happen by, for example, retargeting the styles, or redesigning the buttons to be closer to the design language of `@wordpress/components`.